<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>employeepayment</class>
 <widget class="QMainWindow" name="employeepayment">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1023</width>
    <height>723</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(45, 45, 45);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>471</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Orbitron</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #fff</string>
    </property>
    <property name="text">
     <string>Employee Salary payment</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelbtn">
    <property name="geometry">
     <rect>
      <x>624</x>
      <y>760</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Orbitron</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: rgb(191, 191, 191);
	background-color: rgb(80, 80, 80);
}
QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: rgb(170, 85, 255);
}
</string>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addbtn">
    <property name="geometry">
     <rect>
      <x>844</x>
      <y>760</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Orbitron</family>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: rgb(191, 191, 191);
	
	background-color: rgb(80, 80, 80);
}
QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: rgb(170, 85, 255);
}</string>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>831</width>
      <height>481</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>300</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>600</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Orbitron</family>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>25</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="horizontalSpacing">
      <number>6</number>
     </property>
     <property name="verticalSpacing">
      <number>0</number>
     </property>
     <item row="1" column="1">
      <widget class="QLineEdit" name="empIDtxt">
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>600</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255)</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>600</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>800</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Orbitron</family>
         <pointsize>7</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #fff</string>
       </property>
       <property name="text">
        <string>Type Employee Id to Add Payment</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="empSalarytxt">
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>600</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255)</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="empPayStatuslb">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Orbitron</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #fff</string>
       </property>
       <property name="text">
        <string>Salary Pay Status:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="empsalarylb">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Orbitron</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #fff</string>
       </property>
       <property name="text">
        <string>Employee Salary:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="empPayStatustxt">
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>600</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255)</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="empPayDatelb">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Orbitron</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #fff</string>
       </property>
       <property name="text">
        <string>Pay Date:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="empIDlb">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Orbitron</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #fff</string>
       </property>
       <property name="text">
        <string>Employee ID:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="empPayDatetxt">
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>600</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255)</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="updatebtn">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>600</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Orbitron</family>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: rgb(191, 191, 191);
	
	background-color: rgb(80, 80, 80);
}
QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: rgb(170, 85, 255);
}</string>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelbtn_2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>600</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Orbitron</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: rgb(191, 191, 191);
	background-color: rgb(80, 80, 80);
}
QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: rgb(170, 85, 255);
}
</string>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1023</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelbtn_2</sender>
   <signal>clicked()</signal>
   <receiver>employeepayment</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>616</x>
     <y>612</y>
    </hint>
    <hint type="destinationlabel">
     <x>491</x>
     <y>637</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
